src/components/application_manager/test/commands/hmi/vr_is_ready_response_test.cc

```diff
@@ -73,7 +73,8 @@ class VRIsReadyResponseTest : public CommandsTest<CommandsTestMocks::kIsNice> {
 };
 
 TEST_F(VRIsReadyResponseTest, RUN_SUCCESS) {
-  MessageSharedPtr command_msg(CreateMessage(smart_objects::SmartType_Map));
+  MessageSharedPtr command_msg(
+      CreateMessage(NsSmartDeviceLink::NsSmartObjects::SmartType_Map));
   (*command_msg)[strings::msg_params][strings::number] = kStringNum;
```

Comment:
@wolfylambova22 Also please add `using namespace strings` into your test cases. Therefore you can rewrite
`(*command_msg)[strings::params][strings::connection_key] = kConnectionKey;` to `(*command_msg)[params][connection_key] = kConnectionKey;`